openpype/hosts/houdini/plugins/create/create_pointcache.py

```diff
@@ -37,13 +34,50 @@ def create(self, subset_name, instance_data, pre_create_data):
         }
 
         if self.selected_nodes:
-            parms["sop_path"] = self.selected_nodes[0].path()
+            selected_node = self.selected_nodes[0]
+
+            # Although Houdini allows ObjNode path on `sop_path`rop node
+            #  However, it's preferred to set SopNode path explicitly
+            # These checks prevent using user selecting
+
+            # Allow sop level paths (e.g. /obj/geo1/box1)
+            # but do not allow other sop level paths when
+            #   the parent type is not 'geo' like
+            #   Cameras, Dopnet nodes(sop solver)
+            if isinstance(selected_node, hou.SopNode) and \
+                selected_node.parent().type().name() == 'geo':
```

Comment:
Technically a sop path within a `subnet` would be fine too, right?

Reply:
Technically, yes 

these examples will work 
1. `/obj/obj_test/sop_subnet` 
2. `/obj/obj_test/sop_subnet/output0`
3. `/obj/obj_subnet/obj_box/sop_box`